cmd/controller-manager/app/options/options.go

```diff
@@ -60,6 +60,9 @@ type ServerOption struct {
 	EnableHealthz      bool
 	// To determine whether inherit owner's annotations for pods when create podgroup
 	InheritOwnerAnnotations bool
+	// ConcurrentPGSyncs is the number of pod groups that are allowed to sync concurrently.
+	// Larger number = faster pg creating, but more CPU (and network) load
+	ConcurrentPGSyncs uint32
```

Comment:
It's better unifiy the naming style for job and podgroup. Here is an example:

// WorkerThreadsForPG is the number of threads syncing podgroup operations
// The larger the number, the faster the podgroup processing, but requires more CPU load.
WorkerThreadsForPG